ArchiSteamFarm/Program.cs

```diff
@@ -307,6 +308,16 @@ internal static class Program {
 			}
 		}
 
+		if (InputCryptkeyManually) {
+			string? cryptkey = await Logging.GetUserInput(ASF.EUserInputType.Cryptkey).ConfigureAwait(false);
```

Comment:
You're not doing anything with that `cryptkey`, you know? :3

Reply:
I just re-opened Rider and I still had my cursor on the method declaration of `ArchiCryptoHelper.SetEncryptionKey`. I'll take it as a hint from my body to go to bed.